### ISSUE TITLE
NullRefrenceException when running off the command line

### ISSUE THREAD

**Issue by jbzdarkid** [csaboka/spacechempatch#2]
```
C:\Users\localhost\Downloads>cd SpacechemPatch

C:\Users\localhost\Downloads\SpacechemPatch>SpacechemPatch.exe
Can't find game executable SpaceChem.exe ! Please execute this program from the SpaceChem game folder, or give the path of the game folder as a command line argument. Exiting...

C:\Users\localhost\Downloads\SpacechemPatch>SpacechemPatch.exe "C:\Program Files (x86)\Steam\steamapps\common\SpaceChem"
This program is experimental and was only tested with the Steam version of the game. There is NO WARRANTY on it. Please remember that you can always restore the original game files by verifying game integrity in Steam.
Do you want to continue? [y/N]
Y
Making backup of SpaceChem.exe
Patching...

Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at SpacechemPatch.Program.CollectReplacementsForTypes(IEnumerable`1 typeDefinitions, TypeFinder typeFinder, Dictionary`2 typeReplacements, Dictionary`2 methodReplacements, Dictionary`2 fieldReplacements)
   at SpacechemPatch.Program.CollectReplacements(ModuleDefinition spacechemAssembly, ModuleDefinition ownAssembly, Dictionary`2& typeReplacements, Dictionary`2& methodReplacements, Dictionary`2& fieldReplacements)
   at SpacechemPatch.Program.Main(String[] args)
```

**Comment by jbzdarkid** [comment]
Verified contents and re-ran:
```
This program is experimental and was only tested with the Steam version of the game. There is NO WARRANTY on it. Please remember that you can always restore the original game files by verifying game integrity in Steam.
Do you want to continue? [y/N]
Y
Patching...
All OK
```

Am I supposed to be offered options, or does this just apply "everything"?

**Comment by 12345ieee** [comment]
What release are you using? 0.2 doesn't even have a CLI anymore.

**Comment by jbzdarkid** [comment]
Ah, might've been an older version. Let me try again.

**Comment by jbzdarkid** [comment]
![image](https://user-images.githubusercontent.com/5459085/43824065-d4ba0bcc-9aa5-11e8-8003-5d3890a4a6d4.png)
Or maybe you're not using that. Is there another way to get the version? `--version` maybe?

**Comment by jbzdarkid** [comment]
At any rate, if there's no console version than it must be older, I redownloaded and it seems to be working now.

**Comment by 12345ieee** [comment]
Versions older than 0.2 had the version mistakenly set as 1.0, because it was never changed from the default.

From 0.2 onward, we're keeping the version number right, if you look up the version of the download, you'll see it's `0.2.0.0`.
